source/assets/public/index.css

```diff
@@ -470,7 +453,6 @@ strong {
   }
 
 .c-sponsor__name {
-  font-family: var(--font-subtitle);
```

Comment:
`c-sponsor__name` isn't actually used at the moment, but it might be if we get a sponsor where their logo doesn't have their name visible